src/Moryx.Drivers.OpcUa/OpcUaDriver.cs

```diff
@@ -314,7 +313,7 @@ private async Task<bool> CreateSession(bool firstTry)
         {
             if (firstTry)
             {
-                Logger.Log(LogLevel.Error, "Failed to connect {Uri} ({Message})", builder?.Uri.ToString() ?? OpcUaServerUrl, e.Message);
+                Logger?.Log(LogLevel.Error, "Failed to connect {Uri} ({Message})", builder?.Uri.ToString() ?? OpcUaServerUrl, e.Message);
```

Comment:
OpcUaDriver is a container component and the logger is required. Null-Checks are not necessary!